d2format/format.go

```diff
@@ -306,6 +323,25 @@ func (p *printer) _map(m *d2ast.Map) {
 		prev = n
 	}
 
+	boards := []d2ast.MapNodeBox{}
+	boards = append(boards, layerNodes...)
+	boards = append(boards, scenarioNodes...)
+	boards = append(boards, stepNodes...)
+
+	// draw board nodes
+	for i := 0; i < len(boards); i++ {
+		n := boards[i].Unbox()
+		// if this board is the very first line of the file, don't add an extra indent
+		if n.GetRange().Start.Line != 0 {
```

Comment:
check column == 0 too. who knows, maybe someone wants to 1-line everything

Reply:
sorry, my comment shouldn't and instead should say newline.